Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM public.ecr.aws/bitnami/golang
+FROM golang:1.21.1-alpine3.18
```

Comment:
alpine will not work as `apt` is used for dependencies, use default `golang:latest` or with just a version

Reply:
> alpine will not work as apt is used for dependencies (some packages might not present in alpine also)

checking that only

Reply:
found all the dependencies in apk